### DISCUSSION TITLE
What's about H1 (#) headers?

### DISCUSSION THREAD

**Discussion by CsatariGergely** [google/docsy#2039] [General]
I've realized that a Hugo/Docsy setup does not render the H1 (#) headings into the page navigation menu. One example is [here](https://docs.nephio.org/docs/guides/install-guides/explore-nephio-free5gc/). I've also recognised that Goldydocs does not use H1 headers in the content sections. Should we ignore the usage of H1 headers?

**Comment by fekete-robert** [comment]
In general, Docsy renders the title from the frontmatter as H1, so you shouldn't use it in the content (having multiple H1 headings in a page is supposed to be bad for SEO)

**Reply by CsatariGergely** [reply]
Okey, it is clear now. Köszi
